packages/obonode/obojobo-chunks-youtube/youtube-player.js

```diff
@@ -94,6 +131,146 @@ class YouTubePlayer extends React.Component {
 		)
 	}
 
+	onStateChange(playerState) {
+		const currentPlayheadPosition = Math.floor(this.player.getCurrentTime())
+		const previousSecondsWatched = this.currentState.secondsWatched
+
+		this.previousTime = this.currentTime
+		this.currentTime = Date.now() / 1000
+		const currentSecondsWatched = previousSecondsWatched + (this.currentTime - this.previousTime)
```

Comment:
I want to do this calculation in media store instead of here. That way you can send media store events like play, pause, etc, and it'll figure out how much time has been spent watching.

I also noticed a bug - When changing the page there is a media:unload event but the seconds watched still gets updated with unexpectedly big values